timify-ui/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,55 @@
+import './Footer.css';
+
+const Footer = () => {
+  return (
+    <>
+      <footer className="footer">
+      <div className='footer-wrapper'></div>
+        <div className="footer-section">
+          <div>
+              <h4>Company</h4>
```

Comment:
same here. why do we have things like a careers page, blog, etc?

Reply:
yeah it's the footer from my student store. i forgot to mention in my resources. i was kind of just doing placeholder stuff but i can write the actual things now

Reply:
ah ok. next time include a little more info so we understand the source.